easybuild/easyconfigs/b/BOINC/BOINC-7.16.5-foss-2019b-client.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'MakeCp'
+
+name = 'BOINC'
+version = '7.16.5'
+versionsuffix = "-client"
+
+homepage = 'https://boinc.berkeley.edu'
+description = """BOINC is a program that lets you donate your idle computer time to science projects
+ like SETI@home, Climateprediction.net, Rosetta@home, World Community Grid, and many others."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+sources = [{
+    'filename': '%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/BOINC',
+        'repo_name': 'boinc',
+        'tag': 'client_release/%(version_major_minor)s/%(version)s',
+    },
+}]
+checksums = ['d8c808682d0549b7e1045f1b4633aeb8287619014effc4c16c7b4145b089f988']
+
+dependencies = [
+]
```

Comment:
Old version had cURL as a dependency; doesn't that still apply?

We should also list openssl as os dep
```
osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```

Reply:
What's the policy regarding whether something should be considered a dependency?  The binaries created are linked against a whole bunch of libraries, including curl, but also stuff like
libldap-2.4.so.2 and libbz2.so.1.

Reply:
Some exceptions when we want the OS to provide security updates. OpenSSL falls into this category. I asked the other maintainers, and they though we can add openldap into that category as well. So, please add a `('openldap-devel', 'libldap2-dev')` (or possible disable ldap if it is optional and not important for cluster use, I'm not sure if it is). I don't know what the package names are in ubuntu, `s

For bzip2, cURL , there are packages, and we should definitely link against those.